excluded_urls

```diff
@@ -0,0 +1 @@
+http://localhost:8000/errors/404 https://github.com/uswds/uswds/tree/develop/src/stylesheets/theme http://localhost:8025 https://github.com/usdoj-crt/crt-portal/blob/develop/crt_portal/cts_forms/test_all_section_assignments.py https://fonts.googleapis.com https://fonts.gstatic.com https://www.justice.gov/usao/find-your- https://www.lsc.gov/find- https://www.fbi.gov/contact- http://localhost:4566 http://clamav-rest-dev.apps.internal:9000/scan
```

Comment:
it's good practice in unix to have a newline at the end of the file

https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files